### PR TITLE
build(windows): Do not try to build minizip before zlib.

### PR DESCRIPTION
**Pull request by SlawekNowy**

### CODE REVIEW

**Comment by doug-walker** [comment]
@SlawekNowy , thanks for the PR. Would you please provide more detail about what scenarios require this fix? In other words, most Windows builds succeed already without this, so it must be something more specific about the way you are building.

**Comment by SlawekNowy** [comment]
This is the local windows build. Fetched the repository and built it using Visual Studio generator. Both minizip and zlib are managed by opencolorio. Visual studio 2022 Community x64.

**Comment by SlawekNowy** [comment]
Oh. I forgot to mention I run `--parallel` at cmake build step.

Also it seems that did not fix my problems?

**Comment by SlawekNowy** [comment]
The build is done using `cmake ..` from `build` directory (from ocio dir), and then building generated project.

**Comment by SlawekNowy** [comment]
I'll admit this is not standard setup, but this repo's CMakeLists does fetch deps...
And besides that is standard setup on Linux...

**Comment by SlawekNowy** [comment]
From parent project's discord:
>ok cmake. You get a race condition without the deps' root set, but you don't if I do set the root directories of deps? what?

File: share/cmake/modules/FindExtPackages.cmake [hunk 1]
```diff
@@ -114,6 +114,10 @@ ocio_handle_dependency(  minizip-ng REQUIRED ALLOW_INSTALL
                          MIN_VERSION 3.0.6
                          RECOMMENDED_VERSION 3.0.7
                          RECOMMENDED_VERSION_REASON "Latest version tested with OCIO")
+#minizip-ng requires zlib headers, so we force the dep.
+if(TARGET ZLIB::ZLIB AND TARGET MINIZIP::minizip-ng)
+add_dependencies(MINIZIP::minizip-ng ZLIB::ZLIB)
+endif()
```
Comment (doug-walker) [inline share/cmake/modules/FindExtPackages.cmake:120]: @SlawekNowy , Stephen Yee's suggestion was to revert the changes here in FindExtPackages.cmake since it should be unnecessary after the change to Installminizip-ng.cmake.

**Review by doug-walker — CHANGES_REQUESTED** [review]

**Comment by doug-walker** [comment]
@SlawekNowy , thanks for the update, the code looks perfect now, but unfortunately it looks like you did not sign your latest commit?  The DCO sign-off check needs to pass before we're able to merge it.    Usually "git commit --amend -s" and then re-pushing will fix it.

**Comment by SlawekNowy** [comment]
Closing in favor of #1986.
